cmd/image-builder/main.go

```diff
@@ -477,6 +471,15 @@ func appendMissing(target *map[string]string, source []tags.Tag) {
 	}
 }
 
+// appendMissing appends key, values pairs from source array to target map
+func appendToTags(target *[]tags.Tag, source map[string]string) {
+	if len(source) > 0 {
```

Comment:
I think this is not a mandatory check, because if source is empty then it will not execute at all